Bangazon/Postman/Bangazon.postman_collection.json

```diff
@@ -103,34 +159,31 @@
 					"raw": ""
 				},
 				"url": {
-					"raw": "http://localhost:44372/api/Customers/customers?includes=q",
-					"protocol": "http",
+					"raw": "https://localhost:44372/api/Customers/cusotomers?includes=payments",
+					"protocol": "https",
 					"host": [
 						"localhost"
 					],
 					"port": "44372",
 					"path": [
 						"api",
 						"Customers",
-						"customers"
+						"cusotomers"
```

Comment:
it shows here that you added the incorrect path. The one you addded was cusotomers when it should have been customers I think